bern2/run_bern2.py

```diff
@@ -351,27 +351,17 @@ def generate_base_name(self, text):
     def ner(self, pubtator_file, output_mtner, base_name) -> dict:
         # Run neural model
         start_time = time.time()
-        argparser = argparse.ArgumentParser()
-        argparser.add_argument('--seed', type=int, help='random seed for initialization', default=1)
-        argparser.add_argument('--max_seq_length', type=int,
-                               help='The maximum total input sequence length after tokenization. Sequences longer than this will be truncated, sequences shorter will be padded.',
-                               default=128)
-        argparser.add_argument('--mtner_home', help='biomedical language model home')
-        argparser.add_argument('--disease_only', help='use disease only NER', type=bool, default=True)
-        argparser.add_argument('--time_format', help='time format', default='[%d/%b/%Y %H:%M:%S.%f]')
-        argparser.add_argument("--ner_model_name_or_path", type=str, default="dmis-lab/bern2-ner")
-        argparser.add_argument("--load_model_manually", action="store_true")
-        argparser.add_argument("--s3_bucket", type=str, default="data-science-repository")
-        argparser.add_argument("--local_output", type=str, default="local_output")
-        argparser.add_argument("--use_remote_proxy", action="store_true")
-        argparser.add_argument('--batch_size', type=int, help='The batch size to be run.', default=4)
-        argparser.add_argument("--download_model_bin_file", action="store_true")
-        mt_ner_params = argparser.parse_args()
-        mt_ner_params.model_name_or_path = self.ner_model_name_or_path
-        mt_ner_params.load_model_manually = self.load_model_manually
-        mt_ner_params.use_remote_proxy = self.use_remote_proxy
-        mt_ner_params.batch_size = self.batch_size
-        mt_ner_params.download_model_bin_file = self.download_model_bin_file
+        mt_ner_params = argparse.Namespace(seed=1,
+                                           max_seq_length=128,
+                                           mtner_home=None,
+                                           disease_only=True,
+                                           time_format="dmis-lab/bern2-ner",
+                                           ner_model_name_or_path=self.ner_model_name_or_path or 'dmis-lab/bern2-ner',
```

Comment:
What does this mean (the or thing)?
I think we want to overwrite the default value anyway